realm/src/androidTest/java/io/realm/RealmJsonTest.java

```diff
@@ -689,10 +682,324 @@ private void assertAllTypesPrimaryKeyUpdated() {
         assertEquals(2.23F, obj.getColumnFloat());
         assertEquals(2.234D, obj.getColumnDouble());
         assertEquals(true, obj.isColumnBoolean());
-        assertArrayEquals(new byte[] {1,2,3}, obj.getColumnBinary());
+        assertArrayEquals(new byte[]{1, 2, 3}, obj.getColumnBinary());
         assertEquals(new Date(2000), obj.getColumnDate());
         assertEquals("Dog4", obj.getColumnRealmObject().getName());
         assertEquals(2, obj.getColumnRealmList().size());
         assertEquals("Dog5", obj.getColumnRealmList().get(0).getName());
     }
+
+    // Check the imported object from nulltyps.json[0].
+    private void checkNullableValuesNull(NullTypes nullTypes1) {
```

Comment:
Name seems a bit odd since it also checks the NotNull values? Perhaps just `checkNullableValues`? Ideally it should probably be split into two unit tests?